initrd/bin/change-time.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#change time using hwclock and date -s
+
+clear
+
+echo -e -n "Please enter the current date and time in UTC\n"
+echo -e -n "To find the current date and time in UTC, please check https://time.is/UTC\n\n"
```

Comment:
@JonathonHall-Purism here would be ideal IMO

Reply:
![Screenshot_20240806_140058](https://github.com/user-attachments/assets/f7675870-7089-4ef5-a924-0865c21a1bb2)

Added that, let me know what you think.

I kept the current time reminder in the "TOTP/HOTP mismatch" prompt also because it is relevant to answering the question "do you want to change the time?"

Reply:
Good! You're the picky one on UX, so if you're ok, i'm ok :)